host/sgx/sgxload.c

```diff
@@ -385,15 +410,27 @@ oe_result_t oe_sgx_create_enclave(
     }
     else
     {
-        uint32_t enclave_error;
-        void* base = oe_sgx_enclave_create(
-            NULL, /* Let OS choose the enclave base address */
-            secs->size,
-            enclave_commit_size,
-            ENCLAVE_TYPE_SGX1,
-            (const void*)secs,
-            sizeof(sgx_secs_t),
-            &enclave_error);
+        uint32_t enclave_error = 0;
+        if (ex_features)
+            base = oe_sgx_enclave_create_ex(
+                base_address,
+                secs->size,
+                enclave_commit_size,
+                ENCLAVE_TYPE_SGX1,
+                (const void*)secs,
+                sizeof(sgx_secs_t),
+                (const uint32_t)ex_features,
+                (const void**)ex_features_array,
+                &enclave_error);
+        else
+            base = oe_sgx_enclave_create(
+                base_address,
```

Comment:
Should this be NULL as before?

Reply:
The start_address (later assigned to base_address) is set to NULL in sgxproperties.h if 0-base is not enabled. That start_address is assigned here. Essentially, it will always be NULL in this call.